pom.xml

```diff
@@ -273,6 +273,11 @@
             <version>2.4.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-json</artifactId>
+            <version>2.5.5</version>
```

Comment:
Is this necessary only for tests? Just wondering because I think you mentioned the dependency is via json-path which is scoped for test.
```
<dependency>
            <groupId>com.jayway.jsonpath</groupId>
            <artifactId>json-path-assert</artifactId>
            <version>2.4.0</version>
            <scope>test</scope>
        </dependency>
```

Reply:
Yes, I have just updated it :)

Reply:
Just a thought, I see we're using `rest-assured` in its old repo which bundles with groovy 2.4.4 (https://mvnrepository.com/artifact/com.jayway.restassured/rest-assured/2.9.0). It mentioned its' moved to the following location and that version has groovy 2.4.15 https://mvnrepository.com/artifact/io.rest-assured/rest-assured/3.3.0

Does moving the the latest `rest-assured` fix the problem and avoid us having to manage a transient dependency directly in our pom?

Reply:
That's good spot 👍, will have a look.  I guess dependabot can't do these kind of upgrades.